setup.py

```diff
@@ -107,6 +112,7 @@ def run_tests(self):
     extras_require={
         'flask': flask_requires,
         'tests': tests_require,
+        ":python_version=='2.6'": [contextlib2_that_works_on_py26],
```

Comment:
It's the (semi-undocumented) syntax for environment markers, which is what you need to use if you want wheels to properly do dependencies (otherwise you'd get the `install_requires` of the Python version that ran `pip wheel`).